tests/phpunit/integration/api/fields/includes/class-fields-test-case.php

```diff
@@ -51,11 +51,11 @@ public function setUp() {
 	 * Cleans up the test environment after each test.
 	 */
 	public function tearDown() {
-		parent::tearDown();
-
 		remove_action( 'beans_field_group_label', 'beans_field_label' );
 		remove_action( 'beans_field_wrap_prepend_markup', 'beans_field_label' );
 		remove_action( 'beans_field_wrap_append_markup', 'beans_field_description' );
+
```

Comment:
Why do we need to do the `remove_action` resets before we do the other tear down processes?  What's happening here that warrants rearranging the sequence?

Reply:
Please see my previous comment.